### PR TITLE
feat: "Continue Previous Submission" CTA update

### PR DESCRIPTION
**Pull request by olerichter00**
This PR resolves [ONYX-1030] 

### Description

This PR updates the "Continue Previous Submission" CTA on the `Sell` screen. If there is a draft submission after the user clicks "Exit & Save" in the submission form, this CTA displays the artist and the artist name and let's the user continue the submission.

| Before | After |
| --- | --- |
|Simulator Screenshot - iPhone 15 Pro - 2024-06-03 at 15 22 53 | Simulator Screenshot - iPhone 15 Pro - 2024-06-03 at 15 57 42|

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- "Continue Previous Submission" CTA design update on Sell screen (behind ff) - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[ONYX-1030]: https://artsyproduct.atlassian.net/browse/ONYX-1030?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 77324608259e3f5bba9dc06b72f70aba86c4021b
</p>

File: src/app/Scenes/SellWithArtsy/Components/Header.tsx [hunk 1]
```diff
@@ -1,18 +1,34 @@
...
+export const Header: React.FC<HeaderProps> = (props) => {
   const { draft } = GlobalStore.useAppState((state) => state.artworkSubmission)
+  const { width } = useScreenDimensions()
+
+  const submission = useFragment(FRAGMENT, props.submission)
+
+  const artist = submission?.artist
 
-  const showContinueSubmission = !!enableSaveAndContinue && !!draft?.submissionID
+  const { trackTappedContinueSubmission } = useSubmitArtworkTracking()
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/Components/Header.tsx:31]: This is not yet implemented but I added it here already. Tracking will be implemented in https://github.com/artsy/eigen/pull/10330

File: src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx [hunk 2]
```diff
@@ -158,30 +161,42 @@ const SellWithArtsyHomeContainer = createFragmentContainer(SellWithArtsyHome, {
...
 export const SellWithArtsyHomeScreenQuery = graphql`
-  query SellWithArtsyHomeQuery {
+  query SellWithArtsyHomeQuery($submissionID: ID, $includeSubmission: Boolean = false) {
     recentlySoldArtworks {
       ...SellWithArtsyHome_recentlySoldArtworksTypeConnection
     }
     me {
       ...SellWithArtsyHome_me
     }
+    submission(id: $submissionID) @include(if: $includeSubmission) {
```
Comment (olerichter00) [inline src/app/Scenes/SellWithArtsy/SellWithArtsyHome.tsx:183]: This skips querying for the latest submission if there is no draft.

**Comment by nickskalkin** [comment]
@olerichter00 margin between the "Finish previous submission" section and "Sell art from your collection" title seems too big to me, I think it should be reduced

**Comment by olerichter00** [comment]
> @olerichter00 margin between the "Finish previous submission" section and "Sell art from your collection" title seems too big to me, I think it should be reduced

You are right! I've fixed the spacing, updated the screenshot in the description, and added a condition to display the CTA only if the FF is enabled.

**Review by nickskalkin — APPROVED** [review]
LGTM! 👍

**Pull request merged**